frontend/src/visualization/filtering/ShortestPathMenu.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { Box, Button } from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
+import ShortestPathNodeSelection from './ShortestPathNodeSelection';
+
+/**
+ * Menu where start and end node for shortest path can be selected
+ */
+export default function ShortestPathMenu(): JSX.Element {
+  const [errorNotSpecified, setErrorNotSpecified] = useState(false);
```

Comment:
The variable name is very cumbersome to read. Can you store the opposite, such that the nameing simplifies to `errorSpecified`?